AsyncDisplayKitTests/ASDisplayNodeTests.m

```diff
@@ -2185,8 +2187,10 @@ - (void)testThatBackgroundLayoutSpecOrdersSubnodesCorrectly
     // The inset spec here is crucial. If the nodes themselves are children, it passed before the fix.
     return [ASBackgroundLayoutSpec backgroundLayoutSpecWithChild:overlay background:[ASInsetLayoutSpec insetLayoutSpecWithInsets:UIEdgeInsetsZero child:underlay]];
   };
-  [node layoutThatFits:ASSizeRangeMake(CGSizeMake(100, 100))];
-  node.frame = (CGRect){ .size = node.calculatedSize };
+  
+  ASDisplayNodeSizeToFitSize(node, CGSizeMake(100, 100));
```

Comment:
This looks like a pretty solid group of tests - although most existing, the multiple thread trampoline is a major and helpful addition to the suite and I think we will get meaningful confidence from these!